openvpn/middlewares/server/auth/middleware.go

```diff
@@ -24,6 +24,16 @@ import (
 	"github.com/mysteriumnetwork/go-openvpn/openvpn/management"
 )
 
+const filterLANTemplate = `client-pf %d
+ [CLIENTS DROP]
+ [SUBNETS ACCEPT]
+ -10.0.0.0/8
+ -172.16.0.0/12
+ -192.168.0.0/16
```

Comment:
Protected subnets should be passed from the node. We have a parameter for those.

Reply:
Changed it to pass subnets from outside.